wpilibc/src/main/native/include/frc/Compressor.h

```diff
@@ -79,31 +79,39 @@ class Compressor : public wpi::Sendable,
   bool IsEnabled() const;
 
   /**
-   * Check if the pressure switch is triggered.
+   * Returns the state of the pressure switch.
    *
-   * @return true if pressure is low
+   * @return True if pressure switch indicates that the system is not full,
+   * otherwise false.
    */
   bool GetPressureSwitchValue() const;
 
   /**
-   * Query how much current the compressor is drawing.
+   * Get the current drawn by the compressor.
    *
-   * @return The current through the compressor, in amps
+   * @return Current drawn by the compressor.
```

Comment:
maybe keep the in amps. 
```suggestion
   * @return Current drawn by the compressor in amps.
```

Reply:
That function returns a unit object.

Reply:
The same respective docs in java say "in amps". I get the type but it should probably still be documented.

Reply:
Yes, in Java, the return type is a double, so the unit must be specified. That's not the case in c++, as the unit type system takes care of that.